test/Integration/lower-pack-unpack-without-transpose.mlir

```diff
@@ -0,0 +1,55 @@
+// RUN: tpp-run %s -e big_matmul --entry-point-result=void -print -n 1 --seed=123 2>&1 > %S/lower-pack-unpack-without-transpose-big_matmul-only-default-passes.out
+// RUN: tpp-run --lower-pack-unpack-without-transpose %s -e big_matmul  --entry-point-result=void -print -n 1 --seed=123 2>&1 > %S/lower-pack-unpack-without-transpose-big_matmul-default-passes-lower-pack-unpack-without-transpose.out
+// RUN: fpcmp -r 0.001 %S/lower-pack-unpack-without-transpose-big_matmul-only-default-passes.out %S/lower-pack-unpack-without-transpose-big_matmul-default-passes-lower-pack-unpack-without-transpose.out
+// RUN: rm %S/lower-pack-unpack-without-transpose-big_matmul-only-default-passes.out %S/lower-pack-unpack-without-transpose-big_matmul-default-passes-lower-pack-unpack-without-transpose.out
+
+func.func @big_matmul(%A: tensor<1024x2048xf32>, %B: tensor<2048x4096xf32>, %C: tensor<1024x4096xf32>) -> tensor<1024x4096xf32> {
```

Comment:
That's way too large for CI, single run alone on my local machine takes 25% time of whole test suite.